.vib/suitecrm/cypress/cypress.json

```diff
@@ -0,0 +1,16 @@
+{
+  "env": {
+    "email": "user@example.com",
```

Comment:
It looks like we are not using this env.

Reply:
Indeed, made the change.